### DISCUSSION TITLE
"The folder *.bat doesn't exist" error Opening Winery

### DISCUSSION THREAD

**Discussion by michael-ziedalski** [The-Wineskin-Project/WineskinServer#276] [Q&A] [answered]
Hi, all. So I am trying to create a wrapper in OS X 10.11 (I know it is outdated), and the latest compatible Winery that even starts opening at this version is [Wineskin-2.9.0.7-rc4.app.tar.7z](https://github.com/Gcenx/WineskinServer/blob/51a13480bc352512f136b1de68a37bf856862bc5/Wrapper/Wineskin-2.9.0.7-rc4.app.tar.7z) (from 01/12/21).

It starts opening, but keeps opening not going anywhere, and then when it is closed it displays the following error:

`"Error while writting file: The folder “exe19867523.bat” doesn’t exist."`
<img width="705" alt="bat doesn't exist" src="https://user-images.githubusercontent.com/43689829/177198969-abdf544c-09b0-4f8f-a0d0-88380f8c6bb0.png">

I encountered this once in the past, and the bat file was different, `exec77694335.bat`.

Does anyone know how to get past this error? Is this a bug that was explicitly fixed in some future release? Any advice or help here would be really appreciated.

**Comment by Gcenx** [comment, marked as answer]
You’ll need to downgrade the version of `Wineskin Winery` not just the wrapper.

Outside of the above advice I won’t provide any support for anything below macOS High Sierra.

To have modern wine on legacy versions of macOS/Mac OSX you’ll need to instead build it from source using [macports-wine](https://github.com/Gcenx/macports-wine) my Macports overlay.

**Reply by michael-ziedalski** [reply]
Thanks for helping look into this, Gcenx. 

Would you be able to advise how to downgrade the whole version of `Wineskin Winery`, so that when I download the old wrapper from the repo history, it uses the correct version? I have used AppCleaner previously to delete all new traces of `Wineskin Winery` but it must have missed something - what specifically should I delete to fully downgrade?
